src/main/java/seedu/address/model/expense/ExpenseDate.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.expense;
+
+import java.time.LocalDate;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class ExpenseDate {
+
+
+    //public static final String VALIDATION_REGEX = "^\\d{4}\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])";
+    public static final String VALIDATION_REGEX = "\\d+";
```

Comment:
i think a date format constrained regrex would be more appropriate

Reply:
The regex I tried was not working, shall try again